src/app/modals/user/registered-form.view.ts

```diff
@@ -97,9 +97,14 @@ export const manageIdentityView = {
         },
     ],
     onclick: () => {
+        const platformEnvironment = window.location.href.includes(
+            `platform.int.youwol`,
+        )
+            ? ''
+            : '.int'
         window
             .open(
-                'https://platform.int.youwol.com/auth/realms/youwol/account/#/',
```

Comment:
I believe Julien wanted to always redirect in prod, if this is the case this logic is not necessary. You can double check with him.

Reply:
I confirm that we should always redirect to PROD’s keycloak instance. Also, there is no need for the fragment part of the URL, this is the front app own routing logic. So the URL to unconditionally redirect to is: `https://platform.youwol.com/auth/realms/youwol/account/`